base/linalg/symmetric.jl

```diff
@@ -117,8 +117,16 @@ A_mul_B!{T<:BlasComplex,S<:StridedMatrix}(C::StridedMatrix{T}, A::StridedMatrix{
 *(A::HermOrSym, B::HermOrSym) = full(A)*full(B)
 *(A::StridedMatrix, B::HermOrSym) = A*full(B)
 
-factorize(A::HermOrSym) = bkfact(A.data, symbol(A.uplo), issym(A))
+bkfact(A::HermOrSym) = bkfact(A.data, symbol(A.uplo), issym(A))
+factorize(A::HermOrSym) = bkfact(A)
```

Comment:
That polyalgorithm isn't quite right, because Bunch Kaufman is a better, more general and useful choice than Cholesky on inputs that are symmetric or hermitian but not known to be (semi)definite. It hasn't been implemented for as many types in Julia and it can be a little bit slower for inputs that are known to be (semi)definite, but that's not what the annotation types in Julia represent.